packages/console/src/components/TextInput/index.tsx

```diff
@@ -64,6 +98,7 @@ function TextInput(
       <div
         className={classNames(
           styles.container,
+          hideContentRef && styles.hideTextContainerContent,
```

Comment:
And there are some naming issues. For example, `aContentRef` usually indicates the `ref` object in React, instead of a simple boolean flag. And `hideContent` is a V-O phrase, which is more like a function name instead of a boolean variable or CSS class name